test/fixtures.ts

```diff
@@ -36,16 +36,26 @@ const js = {
 			];
 		})();
 
-		export const esnext = (() => {
+		export const es2021 = (() => {
 			let a;
+			let x = 1;
+			const y = 3;
+			x &&= y;
+			return [
+				a ??= 2,
+				x,
+				1_000_000_000,
```

Comment:
This should be in `esnext` as per [esbuild docs](https://esbuild.github.io/content-types/#javascript).